fdbserver/include/fdbserver/StorageMetrics.actor.h

```diff
@@ -152,15 +152,23 @@ struct StorageServerMetrics {
 	static void add(KeyRangeMap<int>& map, KeyRangeRef const& keys, int delta);
 };
 
-// Contains information about whether or not a key-value pair should be included in a byte sample
-// Also contains size information about the byte sample
+// Contains information about whether or not a key-value pair should be included in a byte sample.
+//
+// The invariant holds:
+//    probability * sampledSize == size
 struct ByteSampleInfo {
+	// True if we've decided to sample this key-value pair.
 	bool inSample;
 
-	// Actual size of the key value pair
+	// Actual size of the key value pair.
 	int64_t size;
 
-	// The recorded size of the sample (max of bytesPerSample, size)
+	// Probability that the key-value pair will be sampled.
+	// This is a function of key and value sizes.
+	// The goal is to sample ~1/250th of the key-value space.
```

Comment:
```suggestion
	// The goal is to sample ~1/BYTE_SAMPLING_FACTOR of the key-value space.
```

Reply:
I don't know. This is a case where it's a lot clearer to me knowing the fraction we're sampling, versus indirecting behind "BYTE_SAMPLING_FACTOR" which took a solid day of reverse-engineering to figure out.

Reply:
It's easy enough to search the codebase and see what value we choose for `BYTE_SAMPLING_FACTOR`, but if you want to include the 250 you could mention its default like you did later. I think it's better, though, that we point to how we actually determine the value. If it's not clear to you that this is a knob, you could also prefix it with `SERVER_KNOBS->BYTE_SAMPLING_FACTOR`